[ADD] MRP: One-step manufacturing (#5158)
odoo/documentation: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst

After completing all of the work orders, click :guilabel:`Produce All` at the top of the screen to
mark the |MO| as :guilabel:`Done`, and register the manufactured product(s) into inventory.

.. image:: one_step_manufacturing/produce-all.png
   :align: center
   :alt: The Produce All button on a manufacturing order.

Tablet view workflow
--------------------

To complete the work orders for an |MO| using the tablet view, begin by navigating to
:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select a manufacturing
order.

Next, click on the :guilabel:`Work Orders` tab, and then select the :guilabel:`📱 (tablet)` button
on the line of the first work order to be processed. This opens the tablet view.

.. image:: one_step_manufacturing/tablet-view-button.png
   :align: center
   :alt: The tablet view button for a work order on a manufacturing order.

After opening the tablet view, Odoo *Manufacturing* automatically starts a timer that keeps track of
how long the work order takes to complete. After completing the work order, click the
:guilabel:`Mark as Done` button in the top-right corner of the tablet view.

> .. image:: one_step_manufacturing/mark-as-done.png

Reviewer: same here
would consider removing

   :align: center
   :alt: The Mark as Done button in the manufacturing tablet view.

Clicking :guilabel:`Mark as Done` while there is at least one more work order left to complete opens
a page that lists the next work order. Click on that work order to open it in the tablet view.

Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done` and register the
manufactured product(s) into inventory.

> .. image:: one_step_manufacturing/done-and-close.png

Reviewer: this isn't very useful without the surrounding UI as navigational context

======================
One-step manufacturing
======================

.. _manufacturing/management/one_step_manufacturing:
.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`

> Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
> one-step manufacturing, Odoo creates a manufacturing order, but does not generate transfers for the
> movement of components out of inventory or finished products into stock. Inventory counts still
> update based on the number of components used and products manufactured, but the act of transferring
> them to and from inventory is not tracked.

Reviewer 2: ```suggestion
Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps.

When using one-step manufacturing, Odoo creates a manufacturing order, but does **not** generate
transfers for the movement of components out of inventory, or finished products into stock.

Inventory counts still update based on the number of components used and products manufactured, but
the act of transferring them to and from inventory is **not** tracked.
```

.. tip::
   The number of steps used in manufacturing is set at the warehouse level, allowing for each
   warehouse to use a different number of steps. To change the number of steps used for a specific
   warehouse, begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`,

>    and then select a warehouse from the :guilabel:`Warehouses` screen.

Reviewer 2: ```suggestion
   then select a warehouse from the :guilabel:`Warehouses` screen.
```

   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input

>    field and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick
>    components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
>    store products (3 steps)`.

Reviewer 2: ```suggestion
   field, and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick
   components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
   store products (3 steps)`.
```

:guilabel:`Bill of Material` field, and the :guilabel:`Product` field auto-populates with the
associated product.

After a |BOM| has been selected, the :guilabel:`Components` and :guilabel:`Work Orders` tabs
auto-populate with the components and operations specified on the |BOM|. If additional components or
operations are required for the manufacturing order being configured, add them to the
:guilabel:`Components` and :guilabel:`Work Orders` tabs by clicking :guilabel:`Add a line`.

Process manufacturing order
===========================

A manufacturing order is processed by completing all of the work orders listed under its
:guilabel:`Work Orders` tab. This can be done on the manufacturing order itself, or from the work
order tablet view.

Basic workflow
--------------

To complete work orders from the manufacturing order itself, begin by navigating to
:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select a manufacturing
order.

On the manufacturing order page, select the :guilabel:`Work Orders` tab. Once work begins on the
first work order that needs to be completed, click the :guilabel:`Start` button for that work order.

> Odoo *Manufacturing* then starts a timer that keeps track of how long the work order takes to
> complete.

Reviewer 2: ```suggestion
Odoo *Manufacturing* then starts a timer to keep track of how long the work order takes to complete.
```
